src/layouts/index.jsx

```diff
@@ -144,6 +145,35 @@ const Layout = ({ children, location }) => {
         </div>
         <Sidebar location={location} />
       </div>
+      <CookieConsent
+        className="mama"
+        disableStyles="true"
+        location="bottom"
+        buttonText="I agree"
+        declineButtonText="Decline"
+        cookieName="gatsby-gdpr-google-analytics"
+      >
+        <p>
+          This website uses cookies to ensure you get the best browsing
+          experience. By continuing to use this website, you consent to our use
+          of these cookies.{" "}
+          <span>
+            This website contains proprietary content that belongs to LivePerson
+            and that is intended for educational use by our customers and
+            prospects. Your use of this site is subject to our{" "}
+            <a
+              href="https://www.liveperson.com/policies/apitou/"
+              target="_blank"
+              rel="noreferrer"
+              aria-label="facebook icon"
+            >

```

Comment:
This file (or changed parts of it): If this isn’t a KC convention, please don’t break tags like this, with each attribute/value pair and each delimiter on separate lines.

(Suggestion: Do you want to come with a first set of guidelines for this? Shape our standards.)